src/lexer/lex.c

```diff
@@ -24,3 +24,35 @@ int unlex(Lexer *l, Token *t) {
     l->unlexed_count++;
     return 0;
 }
+
+int skip_to_token(Lexer *l) {
+    char cur;
+    char prev;
+    if ((cur = fgetc(l->fp)) != EOF) {
+        prev = cur;
+        if (!(cur == ' ' || cur == '/')) {
+            fseek(file, -1, SEEK_CUR);
+            return 0; // token begins immediately
+        }
+    } else {
+        return -1; // file done, no more tokens
+    }
+    // read each character from the file until EOF
```

Comment:
I'm having a bit of trouble understanding what this section of code (until the end of the file) is doing, to be honest. Does this correctly handle a case like
```c
/* Comment
number one*/ /* Comment number two*/ int x = 3;
```

Reply:
That is just a check to make sure the file is not already over. If EOF is the next character, then it returns -1 meaning there are no tokens to get.